### PR TITLE
zh-translation:policy-enforcement/rate-limit/index.md

### PR DESCRIPTION
**Pull request by hueifeng**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by kebe7jun** [comment]
lint failed, please fix it first.

**Comment by hueifeng** [comment]
> lint failed, please fix it first.

@kebe7jun already fixed

File: content/zh/docs/tasks/policy-enforcement/rate-limit/index.md [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+## 开始之前{#before-you-begin}
+
+1. 在Kubernetes集群中安装Istio
+   [Installation Guide](/zh/docs/setup/getting-started/)。
+
+1. 部署[Bookinfo](/zh/docs/examples/bookinfo/)示例应用程序。
+
+## 限制速率{#rate-limits}
+
+Envoy支持两种速率限制:全局和本地。全局速率使用全局gRPC速率限制服务为整个网格提供速率限制。
```
Comment (kebe7jun) [inline content/zh/docs/tasks/policy-enforcement/rate-limit/index.md:25]:
```suggestion
Envoy支持两种速率限制：全局和本地。全局速率使用全局gRPC速率限制服务为整个网格提供速率限制。
```

File: content/zh/docs/tasks/policy-enforcement/rate-limit/index.md [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+虽然入口网关的全局速率限制将对`productpage`服务的请求限制在1 req/min，`productpage`实例的本地速率限制允许10 req/min。
+
+为了确认这一点，从`ratings`pod发送内部`productpage`请求，使用下面的`curl`命令:
+
+{{< text bash >}}
+$ kubectl exec "$(kubectl get pod -l app=ratings -o jsonpath='{.items[0].metadata.name}')" -c ratings -- curl -sS productpage:9080/productpage | grep -o "<title>.*</title>"
+<title>Simple Bookstore App</title>
+{{< /text >}}
+
+你应该看到每个`productpage`实例的请求次数不超过10 req/min。
```
Comment (kebe7jun) [inline content/zh/docs/tasks/policy-enforcement/rate-limit/index.md:340]: 您。

File: content/zh/docs/tasks/policy-enforcement/rate-limit/index.md [hunk 3]
```diff
@@ -0,0 +1,340 @@
...
+
+{{< tip >}}
+`$GATEWAY_URL` is the value set in the [Bookinfo](/zh/docs/examples/bookinfo/) example.
+{{< /tip >}}
+
+您将看到第一个请求通过，但随后的每个请求在一分钟内将得到429响应。
+
+### 验证本地速率{#verify-local-rate-limit}
+
+虽然入口网关的全局速率限制将对`productpage`服务的请求限制在1 req/min，`productpage`实例的本地速率限制允许10 req/min。
```
Comment (kebe7jun) [inline content/zh/docs/tasks/policy-enforcement/rate-limit/index.md:331]: req/min 可以翻译成请求每分。

**Pull request merged**